lib/katello/tasks/upgrades/4.19/enable_structured_apt_for_deb.rake

```diff
@@ -0,0 +1,81 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.19' do
+      desc "Migrate all APT repo content using simple mode to use structured APT mode."
+      task :enable_structured_apt_for_deb => ['environment', 'dynflow:client', "check_ping"] do
```

Comment:
Can we skip as much of this upgrade task as possible if the user has no Debian repos? Perhaps a check on `::Katello::Repository.deb_type.present?` right up front.

Reply:
Made it so that if there is no deb content on the instance the rake task will put a single line of output and then exit:
```
Since there are no deb type repos, enabling structured APT does not require migration. Skipping.
```